operator/database/kinds/databases/managed/adapt.go

```diff
@@ -264,10 +273,43 @@ func Adapter(
 					internalMonitor.Info("set current state of managed database")
 				}
 
+				backupDefs := map[string]*tree.Tree{}
+				if desiredKind.Spec.Backups != nil {
+					backupDefs = desiredKind.Spec.Backups
+				}
+				cleanupDestroy, err := cleanup(
+					monitor,
+					backupDefs,
+					k8sClient,
+					namespace,
+					getBackupLabels(),
+				)
```

Comment:
into query phase?

Reply:
yes if you look into the function you would see that the kubeapi has to be queried as to which jobs are currently present

Reply:
if i look into the function i can see that it returns destroyers based on a kubeapi query. So it belongs to the query phase, not to the ensure phase?